jenkins-pipelines/rolling-upgrade-alternator.jenkinsfile

```diff
@@ -5,7 +5,7 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 rollingUpgradePipeline(
     backend: 'gce',
-    base_versions: ['2020.1', '4.4', '2021.1'],
+    base_versions: ['2020.1', '4.4', '2021.1', '4.5'],
```

Comment:
Why do we have here also 2020.1 and 2021.1?
Is it related to the auto_base version?
If so, it's weird because it shouldn't be in the jenkins pipeline at all.
If it doesn't, it's also weird because we don't test 2020.1 /2021.1 to 4.5

Reply:
> 

AFAIK this is there because the enterprise branch (in scylla.git) use our SCT master branch (this one), and we need these base versions to upgrade to it... and the OSS versions know to drop the enterprise versions from it (as they are not supported).

once we move to the auto mode, we won't have to set it anymore, as the versions will be found automatically for both enterprise and OSS

Reply:
> AFAIK this is there because the enterprise branch (in scylla.git) use our SCT master branch (when this branch was created), and we need these base versions to upgrade to it... and the OSS versions know to drop the enterprise versions from it (as they are not supported).

That's right reason

Reply:
> Is it related to the auto_base version?

auto_base PR isn't backported to branch-4.5, so we still need to update the base_version

Reply:
Thanks Amos :)